python/pyhie/allspark/model/qwen_v15.py

```
@@ -490,15 +498,75 @@ def _trans_weight(self, torch_weight, lora_name=None):
+        def _load_scale_inv_raw(name):
+            """Load scale_inv tensor for an FP8 weight (no permute)."""
+            if isinstance(name, str):
+                scale_key = name + '_scale_inv'
+                if scale_key in torch_weight:
+                    return torch_weight[scale_key].cpu()
+            return None
+
+        # Dispatch helper: for non-LoRA direct-load, feed to engine;
+        # otherwise write to the weights file.
+        if lora_name:
+            _w = lambda k, t, m, sm, gl: save_torch_to_allsparky(weights_path, k, t, m, sm, gl)
```

Flake8 (7.3.0)

[error] do not assign a lambda expression, use a def

(E731)